spec/spec_helper.rb

```diff
@@ -10,7 +10,7 @@
 require 'voxpupuli/test/spec_helper'
 
 RSpec.configure do |c|
-  c.hiera_config = 'spec/fixtures/hiera.yaml'
+  c.hiera_config = spec/fixtures/hiera.yaml
```

Comment:
In other repos we quote it in YAML:
```console
$ rg hiera_config modules/voxpupuli/*/.sync.yml
modules/voxpupuli/puppet-telegraf/.sync.yml
9:  hiera_config: "'spec/hiera.yaml'"

modules/voxpupuli/puppet-openvpn/.sync.yml
3:  hiera_config: "'spec/fixtures/hiera/hiera.yaml'"
```

I think this is to allow using variables.